### ISSUE TITLE
Error compiling to JavaScript:

### ISSUE THREAD

**Issue by ssnaruka** [dart-lang/dart-pad#1556]
## What happened?
```
    I compile the sample hello world program it will run the first time on the tab and if I try again the same or another code then it can show the error Error compiling to JavaScript: XMLHttpRequest error. When I open a new window in chrome then in this window it runs the first time and then after showing an error. I follow troubleshooting steps but nothing happened. 
```

## Browser

Browser: Google Chrome 

Version: 81.0.4044.138 (Official Build) (64-bit)

Are you using any extensions/plugins that affect website behavior
(particularly those that affect iframes, such as ad blockers)?

```
yes, I use ad blocker
```

Are there any warnings or errors in your browser's JavaScript console?
If so, paste them below:

```
Error compiling to JavaScript:
XMLHttpRequest error.
```

### Machine: 
Dell Inspiron 15

Operating system: Windows 10 Home Single Language

Version: 1903

## Code
```
void main() {
  for (int i = 0; i < 5; i++) {
    print('hello ${i + 1}');
  }
}
```

### DartPad's output
```
Error compiling to JavaScript:
XMLHttpRequest error.
```

**Comment by tplive** [comment]
Getting the same. I see a dev console error message: 
```
Source map error: Error: request failed with status 404
Resource URL: https://dartpad.dev/scripts/playground.dart.js
Source Map URL: playground.dart.js.map
```

**Comment by domesticmouse** [comment]
I believe this is related to https://github.com/dart-lang/dart-pad/issues/1554.

**Comment by NiyazAhamed** [comment]
Getting the same error.

**Comment by allentv** [comment]
I have the same error too.

**Comment by mikeswartz** [comment]
It seems the backend services this is calling might need to allow the request.

something like (example only):

request.headers['Access-Control-Allow-Origin'] = '*';
request.headers['Access-Control-Allow-Methods'] = 'GET, POST, OPTIONS, PUT, DELETE';

**Comment by allentv** [comment]
I am going to check `dart-services` repo to see how things work and if I am able to pinpoint the problem, will create a PR.

**Comment by allentv** [comment]
So I downloaded the `dart-services` repo on my machine and ran the service locally. I made a POST request via Postman to the `analyze` endpoint and can confirm that the CORS headers are available in the response.

<img width="962" alt="Screenshot 2020-05-12 at 21 49 00" src="https://user-images.githubusercontent.com/6534992/81743946-69418f00-949a-11ea-8434-21f47d9a862c.png">

The issue seems to be a problem with the server which seems to be working now.
